src/java.base/share/classes/com/sun/crypto/provider/GaloisCounterMode.java

```diff
@@ -426,6 +426,49 @@ void doLastBlock(byte[] in, int inOfs, int len, byte[] out, int outOfs,
         }
     }
 
+    // Process en/decryption all the way to the last block.  It takes both
+    // For input it takes the ibuffer which is wrapped in 'buffer' and 'src'
+    // from doFinal.
+    void doLastBlock(ByteBuffer buffer, ByteBuffer src, ByteBuffer dst)
```

Comment:
The ordering of these new ByteBuffer-arg methods seems random? Perhaps, either group them altogether or move them so that they are together with the byte[] counterpart methods?